x11-libs/libsm/libsm-1.2.2.recipe

```diff
@@ -38,18 +38,23 @@ REQUIRES_devel="
 
 BUILD_REQUIRES="
 	haiku${secondaryArchSuffix}_devel
+	docbook_xml_dtd
+	docbook_xsl_stylesheets
```

Comment:
`d` is before `h` so...

```diff
-	haiku${secondaryArchSuffix}_devel
	docbook_xml_dtd
	docbook_xsl_stylesheets
+	haiku${secondaryArchSuffix}_devel
```

Reply:
Yep, but I think there is an implicit rule which allows to put `haiku${secondaryArchSuffix}_devel` first.

Reply:
I would prefer haiku on top too ;)

Reply:
Well, it's fine if y'all prefer that :)

Reply:
Again one of those thing we should have a mutual rule for ;)